pkgs/applications/science/math/bcal/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, fetchFromGitHub, python3Packages }:
+
+with stdenv.lib;
+
+stdenv.mkDerivation rec {
+  name = "bcal-${version}";
+  version = "1.7";
+
+  src = fetchFromGitHub {
+    owner = "jarun";
+    repo = "bcal";
+    rev = "v${version}";
+    sha256 = "08cqp2jysvy743gmwpzkbqhybsb49n65r63z3if53m3y59qg4aw8";
+  };
+
+  nativeBuildInputs = [ python3Packages.pytest ];
+
+  doCheck = true;
+  checkPhase = ''
+    python3 -m pytest test.py
+  '';
+
+  installFlags = [ "DESTDIR=$(out)" "PREFIX=" ];
```

Comment:
`makeFlags = [ "CC=cc" ];` hopefully fixes darwin.

Reply:
https://github.com/jarun/bcal/blob/master/Makefile#L1

Reply:
OK, thank you. Let's see if this works. Additionally, If I'm not mistaken the `-m64` flag leaves us only with `x86_64` systems which is why `aarch64` fails. How would I best go on with specifiying this in the meta section?

Reply:
`["x86_64-darwin" "x86_64-linux" ]`

Reply:
But actually `-m64` should be skipped on `aarch64`. This could be fixed upstream.

Reply:
https://github.com/jarun/bcal/issues/17